### ISSUE TITLE
Print out case and control sample size

### ISSUE THREAD

**Issue by Shicheng-Guo** [rgcgithub/regenie#123]
current version (2.0.2) only show the total sample size, without showing case and control sample size separately. Hope next version, the sample size for case and control can be showed. 

```
 * phenotypes       : [/home/sguo2/janssen4/ukb/analytic/UKB_PHENOME_20210218.Guo.vRegenie.txt] n_pheno = 1
   -dropping observations with missing values at any of the phenotypes
   -number of phenotyped individuals = 161590
   **-number of individuals remaining with non-missing phenotypes = 161590**
```

Start time: Sun May 16 20:04:46 2021

              |============================|
              |        REGENIE v2.0.2      |
              |============================|

Copyright (c) 2020 Joelle Mbatchou and Jonathan Marchini.
Distributed under the MIT License.

**Comment by joellembatchou** [comment]
Hi,

Do you mean print out the case/control counts for each trait analyzed also within the log?

**Comment by Shicheng-Guo** [comment]
Yes. Exactly. Thanks.

**Comment by Shicheng-Guo** [comment]
We can do two things:

1, in the log file, we can print case/control sample size
2, in the Regenie association output file, we can output case/control for each sample size
3, in order to decrease the file size, we can output the file as bgz compressed file format.

**Comment by joellembatchou** [comment]
Option (1) is now available in v2.2 using `--verbose`.
